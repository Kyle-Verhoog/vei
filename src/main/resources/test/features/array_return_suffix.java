// FAIL
public class Array_return_suffix {
  public Array_return_suffix() {}
  public int foo()[] { return null; }
}