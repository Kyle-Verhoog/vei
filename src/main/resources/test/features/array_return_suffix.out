PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ foo
( ┃ (
) ┃ )
[ ┃ [
] ┃ ]
{ ┃ {
RETURN ┃ return
NULL_LITERAL ┃ null
; ┃ ;
} ┃ }
} ┃ }